docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1034,7 +1034,7 @@
         <para><ulink
         url="https://github.com/hpcc-systems/HPCC-Platform/blob/master/helm/hpcc/docs/placements.md">https://github.com/hpcc-systems/HPCC-Platform/blob/master/helm/hpcc/docs/placements.md</ulink></para>
 
-        <sect4 id="YAML_FileStruct_Placement">
+        <sect4 id="YAML_FileStruct-Placement">
```

Comment:
I think this should be sect3, and line 1098 should be /sect3.  
If it builds I won't worry though..

Reply:
It builds.  I'm rather certain that entire <sect4 id="YAML_FileStruct-Placement"> ... </sect4> is a duplicate that should be removed. But I'm not certain exactly what parts of it should be removed, which one is the correct one, etc.  Greg made a lot of changes to that specific file and I think the people doing the translation were confused when they saw the diff.